nesting.md

```diff
@@ -66,40 +66,39 @@ api.tx.unique.setCollectionPermissions(ACollectionId, {
   nesting: {
     OwnerRestricted: [BCollectionId, DCollectionId]
   }
 });
 ```
-Now, owners can nest tokens only from `collection B` or `collection D`.
+As a result, the owners would be able to nest tokens only from `collection B` or `collection D`.
 
-Finally, a collection owner can disable nesting at any time by issuing the `setCollectionPermissions` transaction like this:
+Finally, a collection owner can disable nesting at any time by issuing the `setCollectionPermissions` transaction:
 ```javascript
 api.tx.unique.setCollectionPermissions(ACollectionId, {
   nesting: 'Disabled'
 });
 ```
 
-When a collection owner disables nesting, all existing bundles remain, a token owner can unnest nested tokens from any bundle owned by him. But token owners now are not allowed to create new bundles.
+When a collection owner disables nesting, all existing bundles remain, a token owner can unnest nested tokens from any bundle owned by him. Creating new bundles, however, will not be possible.
 
 The nesting rules can be changed at any time, the existing bundles won’t be affected.
 
 ## Additional info
- * You can’t burn NFT bundles, you have to destruct them first (unnest or burn the nested tokens). When the root NFT has no
-   children (i.e., it became a regular NFT), you can burn it as well.
+ * You can’t burn NFT bundles (i.e. entire branches), you have to deconstruct them first (unnest or burn the nested tokens separately). Once the root NFT has no more children (i.e. when it reverts to a regular NFT) it can then be burned it as well.
  * You can’t burn non-empty collections.
 
-## Playgrounds to get started
+## Playgrounds - get started
  * [src/playgrounds/simpleNesting.dev.js](https://github.com/UniqueNetwork/unique-playgrounds/blob/angelhack-polkadot-2022/src/playgrounds/simpleNesting.dev.js)
 this playground shows you how to connect to the endpoint, create a user object from a seed, create a collection and how mint and nest tokens.
  * [src/playgrounds/explicitSimpleNesting.dev.js](https://github.com/UniqueNetwork/unique-playgrounds/blob/angelhack-polkadot-2022/src/playgrounds/explicitSimpleNesting.dev.js)
-pretty much the same playground as the first one, but it shows how nesting works under the hood. You can use regular transfer to nest one token into another.
+pretty much the same as above, but it shows how nesting works under the hood. You can use a regular transfer to nest one token into another.
  * [src/playgrounds/listAccountTokens.dev.js](https://github.com/UniqueNetwork/unique-playgrounds/blob/angelhack-polkadot-2022/src/playgrounds/listAccountTokens.dev.js)
-this playground shows you how you can get all owned tokens of a specific account inside a given collection. This is the point where you can get to know how to traverse an NFT bundle.
+this playground shows you a way to list all owned tokens of a specific account within a given collection. This is all about traversing an NFT bundle.
```

Comment:
I'm not sure that it's "this is all about traversing an NFT bundle".
It's more like a prelude to it.

Reply:
Oh, 'this is all about' is a term that means 'all the content presented here is about' So it can actually be changed to:
'The material presented here is all about NFT bundle traversal.

Reply:
Okay, then we can leave it. Thanks